compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/SchemaValidatorTest.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
check other places as well

Reply:
fixed with [e67b64d](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1238/commits/e67b64dd275c83e33581684174c4aafd9843a20e)